spring-cloud-kubernetes-fabric8-discovery/src/test/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesCatalogWatchTest.java

```diff
@@ -18,6 +18,8 @@
 
 import java.util.Collections;
```

Comment:
I had to slightly change these tests, because on closer inspection they were not testing what they promised they will.